ncurses.go

```diff
@@ -307,3 +307,14 @@ func UseDefaultColors() error {
 func UseEnvironment(use bool) {
 	C.use_env(C.bool(use))
 }
+
+// TabSizeSet allows for modification of the tab width. This setting is global
+// and affects all windows.
+func TabSizeSet(tabSize int) {
+	C.TABSIZE = C.int(tabSize)
+}
+
+// TabSizeSet returns the configured tab width.
+func TabSizeGet() int {
```

Comment:
To match the project style, this should be renamed to:
```suggestion
func TabSize() int {
```

Reply:
Updated accordingly. Thanks for reviewing.